quickstarts/apache-flink/config.yml

```diff
@@ -0,0 +1,40 @@
+
+
+slug: apache-flink
+description: |
+  ## Comprehensive monitoring quickstart for Apache Flink
+  New Relic detects your Apache Flink metrics like job manager (checkpoints, direct, mapped, metaspace memory usage),
+  task manager (shuffle netty, operators), heap memory usage and more.
+
+  ## Why monitor Apache Flink?
+  An ideal Apache Flink Performance Monitoring provides an easy way for effective monitoring and alerting of your Flink jobs.
+
+
+  ## What’s included in this quickstart?
+  New Relic's Apache Flink monitoring quickstart boasts instant full-stack observability out-of-the-box:
+  - High value alerts for your Apache Flink (errors, usage metrics and JVM).
+  - Easy to use performance monitoring dashboards.
+  - Custom alerts that proactively inform developers about the status of their Apache Flink Jobs.
+
+
+summary: |
+  New Relic’s instant observability quickstart with Apache Flink monitoring helps to detect client, job manager and task manager logs and parses them into a JSON payload.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Apache Flink
+documentation:
+  - name: Apache Flink integration documentation
+    description: |
+      New Relic’s instant observability quickstart with Apache Flink monitoring helps to detect client, job manager and task manager logs and parses them into a JSON payload.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-openmetrics/install-apache-flink/
+keywords:
+  - apache-flink
```

Comment:
please add the below also as keywords

- flink
- apache flink
- data stream

Reply:
Added keywords to the above keywords.

Thank you... 😊